source/features/conversation-activity-filter.tsx

```diff
@@ -221,6 +227,7 @@ async function init(): Promise<void> {
 		applyState(state);
 	}
 
+	window.addEventListener('hashchange', uncollapseTargetedComment);
```

Comment:
This PR can be merged as is but in the other open PR we can add an AbortSignal here

Reply:
> This PR can be merged as is but in the other open PR we can add an AbortSignal here

I think the PR is maybe this one (https://github.com/refined-github/refined-github/pull/5523/files/d1511cb12654a480622e643c85f1dec87fef9d23).

Should I add `signal` like the below code in this PR?

```ts
async function init(signal: AbortSignal): Promise<void> {
	const state = minorFixesIssuePages.some(url => location.href.startsWith(url))
		? 'hideEventsAndCollapsedComments' // Automatically hide resolved comments on "Minor codebase updates and fixes" issue pages
		: 'default';

	await addWidget('#partial-discussion-header .gh-header-meta :is(clipboard-copy, .flex-auto)', state);
	await addWidget('#partial-discussion-header .gh-header-sticky :is(clipboard-copy, relative-time)', state);

	if (state !== 'default') {
		applyState(state);
	}

	// To do: when `:has` is available, replace below logic with css (#5390)
	window.addEventListener('hashchange', revealCollapsedComment, {signal});

	document.body.addEventListener('keypress', runShortcut, {signal});
}
```